.gitignore

```diff
@@ -293,4 +293,8 @@ src/GraphControl/GraphingImplOverrides.props
 src/CalcViewModel/DataLoaders/DataLoaderConstants.h
 !/build/config/TRexDefs/**
 !src/Calculator/TemporaryKey.pfx
-!src/CalculatorUnitTests/CalculatorUnitTests_TemporaryKey.pfx
\ No newline at end of file
+!src/CalculatorUnitTests/CalculatorUnitTests_TemporaryKey.pfx
```

Comment:
This was not the change was requested by Eric, it causes these output folders to be ignored.  Otherwise,  the branch, repo, PR, etc would be flooded with changes not needed when pushing a commit.